dev-cpp/benchmark/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
We can't add new packages without maintainers, it's against QA policy. 
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602